implementations/rust/ockam/ockam_transport_ble/src/driver/stream.rs

```diff
@@ -0,0 +1,97 @@
+use ockam_core::compat::sync::Arc;
+
+#[cfg(feature = "std")]
+use futures::lock::Mutex;
+
+#[cfg(not(feature = "std"))]
+use super::mutex::Mutex;
+
+use crate::driver::BleStreamDriver;
+
+pub(crate) struct AsyncStream<A>
+{
```

Comment:
Nit: I think you need to run `cargo fmt`.

Reply:
Whoops :-)